CMakeLists.txt

```diff
@@ -14,17 +15,27 @@ IF(COMMAND CMAKE_POLICY)
   endif(POLICY CMP0022)
 ENDIF(COMMAND CMAKE_POLICY)
 
-PROJECT(WT)
+option(HUNTER_ENABLED "Enable Hunter package manager" OFF)
+if(HUNTER_ENABLED)
```

Comment:
This `if` is redundant:
* https://docs.hunter.sh/en/latest/overview/compatibility.html